CHANGELOG.md

```diff
@@ -7,24 +7,41 @@ be found.
 
 ## 17.03.0-ce (2017-03-01)
 
+**IMPORTANT**: Starting with this release, Docker is on a monthly release cycle and uses a
+new YY.MM versioning scheme to reflect this. In addition to a monthly release, more stable
```

Comment:
I updated PTAL, regarding `How can users distinguish a "monthly" release from a "quarterly" release?` I think we need some sort a graph